### PR TITLE
[UWP] fix so tests are compatible with UWP

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change ###
Just made some small changes so the UI tests are compatible with UWP. 

### PR Checklist ###

- [x] Has automated tests 
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

**Comment by PureWeen** [comment]
build --uitests

**Comment by PureWeen** [comment]
build --uitests

**Comment by samhouts** [comment]
tests don't seem happy

**Comment by PureWeen** [comment]
I rebased to 3.6.0 and kicked off ui tests again
The failing tests being reported were just from github not reporting the results correctly

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue4597.cs [hunk 1]
```diff
@@ -129,6 +138,9 @@ public void TestImagesDisappearCorrectly()
 					}
 				}
 			}
+#else			
+			//can't validate if images have vanished until this is resolved
+			Assert.Inconclusive(@"https://github.com/xamarin/Xamarin.Forms/issues/4731");
```
Comment (paymicro) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue4597.cs]: `Assert.Inconclusive` throws exception.
This test may pass only on android and ios?

Comment (PureWeen) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue4597.cs]: Good point. I marked this test with the category UWPIgnore for now so when we're running the suite of UWP tests it won't run

Comment (paymicro) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue4597.cs]: The manual test still runs and reaches this point, but ends without exception.

**Comment by samhouts** [comment]
None of the failing tests look related. 👍

**Pull request merged**